tests/lax_numpy_test.py

```diff
@@ -1674,6 +1674,25 @@ def testPolySub(self, a_shape, b_shape, dtype):
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}_order={}_k={}".format(
+          jtu.format_shape_dtype_string(a_shape, dtype),
+          order, k),
+       "dtype": dtype, "a_shape": a_shape, "order" : order, "k": k}
+      for dtype in default_dtypes
+      for a_shape in one_dim_array_shapes
+      for order in range(5)
+      for k in [np.arange(order, dtype=dtype), np.zeros(1, dtype), np.ones(1, dtype), None]
```

Comment:
There is one issue here: when `order=0`, `np.zeros(1)` and `np.arange(order)` are identical, which leads to a duplicated test ID and a failure of the test scaffold. Perhaps the easiest thing is to remove `np.zeros()` from the list?